### ISSUE TITLE
flameshot complaining about AF_NETLINK socket

### ISSUE THREAD

**Issue by jose1711** [netblue30/firejail#4925]
### Description

`flameshot` generates the following errors while running:
```
Could not create AF_NETLINK socket (Operation not supported)
```

### Steps to Reproduce

1. Install and run `flameshot` from terminal emulator
2. Watch the terminal output

### Expected behavior

No errors should appear.

### Actual behavior

```
Could not create AF_NETLINK socket (Operation not supported)
Could not create AF_NETLINK socket (Operation not supported)
Could not create AF_NETLINK socket (Operation not supported)
Could not create AF_NETLINK socket (Operation not supported)
Could not create AF_NETLINK socket (Operation not supported)
```
Messages are also written to `systemd`'s journal:
```
$ journalctl -b 0 | grep -c NETLINK
Hint: You are currently not seeing messages from other users and the system.
      Users in groups 'adm', 'systemd-journal', 'wheel' can see all messages.
      Pass -q to turn off this notice.
491
```

### Behavior without a profile

If run with `--noprofile` the issue disappears. Another way is to add `netlink` to protocol list.

### Environment

- Arch Linux x86_64
- Firejail version = current master

### Checklist

- [x] The issues is caused by firejail (i.e. running the program by path (e.g. `/usr/bin/vlc`) "fixes" it).
- [x] I can reproduce the issue without custom modifications (e.g. globals.local).
- [x] The program has a profile. (If not, request one in `https://github.com/netblue30/firejail/issues/1139`)
- [x] The profile (and redirect profile if exists) hasn't already been fixed [upstream](https://github.com/netblue30/firejail/tree/master/etc).
- [x] I have performed a short search for similar issues (to avoid opening a duplicate).
  - [ ] I'm aware of `browser-allow-drm yes`/`browser-disable-u2f no` in `firejail.config` to allow DRM/U2F in browsers.
- [ ] I used `--profile=PROFILENAME` to set the right profile. (Only relevant for AppImages)

**Comment by rusty-snake** [comment]
Is anything broken?

**Comment by jose1711** [comment]
No, it looks to be working just fine.

**Comment by netblue30** [comment]
We disable netlink socket on all sort of programs. Some of them are still trying to access it for whatever reasons. Unless the program functionality is breaking, we'll keep it disabled. Closing the bug for now.

**Comment by kmk3** [comment]
@jose1711 commented [on Feb 9](https://github.com/netblue30/firejail/issues/4925#issue-1129200262):

> ### Expected behavior
> 
> No errors should appear.
> 
> ### Actual behavior
> 
> ```
> Could not create AF_NETLINK socket (Operation not supported)
> Could not create AF_NETLINK socket (Operation not supported)
> Could not create AF_NETLINK socket (Operation not supported)
> Could not create AF_NETLINK socket (Operation not supported)
> Could not create AF_NETLINK socket (Operation not supported)
> ```

I'd suggest reporting this to flameshot; maybe it can be changed to try once at
startup to use netlink and if it doesn't work, ignore it or always try
something else instead (to reduce the amount of error messages).

> Messages are also written to `systemd`'s journal:
> 
> ```
> $ journalctl -b 0 | grep -c NETLINK
> Hint: You are currently not seeing messages from other users and the system.
>       Users in groups 'adm', 'systemd-journal', 'wheel' can see all messages.
>       Pass -q to turn off this notice.
> 491
> ```

Maybe also change it to log this error only once per instance (or not at all).

**Comment by rusty-snake** [comment]
> I'd suggest reporting this to flameshot;

IIRC this is Qt.

**Comment by Pictor13** [comment]
I'm getting this error even after `flameshot` was removed...

**Comment by rusty-snake** [comment]
This is over one year old. Please open a new issue.

And BTW how do you start flameshot if it is removed.

**Comment by Pictor13** [comment]
> And BTW how do you start flameshot if it is removed.

I was wondering as well exactly why I got logs, for a software that was removed.

Apparently a reboot of Ubuntu cleared up dangling processes or binds and now the log doesn't appear anymore.
